### PR TITLE
Patch flux v2 to v0.36.0

### PR DESCRIPTION
**Pull request by ciaranca**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-7175

### Change description ###

Patch flux v2 to v0.36.0

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: apps/flux-system/base/image-automation-components.yaml [hunk 1]
```diff
@@ -1995,7 +2037,7 @@ spec:
     spec:
       containers:
       - args:
-        - --events-addr=http://notification-controller/
```
Comment (timja) [inline apps/flux-system/base/image-automation-components.yaml:1998]: is this intentionally changing?

File: apps/flux-system/base/image-automation-components.yaml [hunk 2]
```diff
@@ -2005,7 +2047,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: hmctspublic.azurecr.io/fluxcd/image-automation-controller:v0.19.0
+        image: ghcr.io/fluxcd/image-automation-controller:v0.26.1
```
Comment (timja) [inline apps/flux-system/base/image-automation-components.yaml:2050]: is this intentionally changing? it should be fine but just want to make sure its intended

File: apps/flux-system/base/image-automation-components.yaml [hunk 3]
```diff
@@ -2070,7 +2120,7 @@ spec:
     spec:
       containers:
       - args:
-        - --events-addr=http://notification-controller/
```
Comment (timja) [inline apps/flux-system/base/image-automation-components.yaml:2073]: is this intentional?

File: apps/flux-system/base/image-automation-components.yaml [hunk 4]
```diff
@@ -2080,7 +2130,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: hmctspublic.azurecr.io/fluxcd/image-reflector-controller:v0.12.0
+        image: ghcr.io/fluxcd/image-reflector-controller:v0.22.1
```
Comment (timja) [inline apps/flux-system/base/image-automation-components.yaml:2133]: is this intentional?

File: apps/flux-system/base/image-automation-components.yaml (hunk 2 above)
Comment (ciaranca) [reply apps/flux-system/base/image-automation-components.yaml]: Spoke with @adusumillipraveen, we were unsure why its imported into registry, checked against sds and they are using source

File: apps/flux-system/base/image-automation-components.yaml (hunk 1 above)
Comment (ciaranca) [reply apps/flux-system/base/image-automation-components.yaml]: Mentioned this to Praveen earlier. I checked sds and not set there. Do you know if it should be? Was testing without it.

**Comment by ciaranca** [comment]
Closing this PR, going to redo in a cleaner manner

**Pull request closed without merging**